janet-async/async-adapter/src/main/java/io/techery/janet/AsyncActionAdapter.java

```diff
@@ -68,6 +68,16 @@ public AsyncActionAdapter(String url, AsyncClient client, Converter converter) {
         return AsyncAction.class;
     }
 
+    @Override protected <A> void cancelInternal(A action) throws JanetException {
```

Comment:
seems like `cancel` is not so important as `send` but comes before (sorting problem)